READMEs/gas-strategy-remote.md

```diff
@@ -0,0 +1,57 @@
+<!--
+Copyright 2023 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Use Specific Gas Strategy for Remote Networks
+
+This quickstart illustrates the definition of gas strategy in ocean-lib stack in order to
+confirm the transactions on blockchain as soon as possible in case the network is congested.
+
+Here are the steps:
+
+1.  Setup
+2.  Define gas strategy
+3.  Alice publishes the asset using gas strategy
+
+Let's go through each step.
+
+## 1. Setup
+
+Ensure that you've already (a) [installed Ocean](install.md), and (b) [set up remotely](setup-remote.md).
+
+## 2. Define gas strategy
+
+Fees are defined for `polygon-main` & `polygon-test` networks.
+
+```python
+from tests.integration.remote.util import get_gas_fees_for_remote
+
+priority_fee, max_fee = get_gas_fees_for_remote()
+```
+
+## 3. Alice publishes the asset using gas strategy
```

Comment:
I would also make it clear that these can be added to any tx_dict, and this is just an example